doc/system-upgrade.rst

```diff
@@ -24,8 +24,8 @@ Description
 -----------
 
 ``dnf system-upgrade`` can be used to upgrade a Fedora system to a new major
-release. It replaces fedup (the old Fedora Upgrade tool).
-
+release. It replaces fedup (the old Fedora Upgrade tool). Before you proceed ensure that your system
+is fully upgraded (``dnf upgrade --refresh``).
```

Comment:
It should be `dnf --refresh upgrade`. Technically the order doesn't matter, but that's how the man page displays it, so we should be consistent.